storhaug

```diff
@@ -266,26 +274,31 @@ EXPORT {
   FSAL {
     Name = "GLUSTER";
     Hostname = localhost;
-    Volume = "${1}";
+    Volume = "${volume}";
   }
 }
 EOF
 
-    exp_id=$(ls -1 /run/gluster/shared_storage/nfs-ganesha/exports/export.*.conf | wc -l)
+    exp_id=$(find /run/gluster/shared_storage/nfs-ganesha/exports -name 'export.*.conf' | wc -l)
```

Comment:
I didn't test your code but did you think about this issue #26 ?

Reply:
Nope, hadn't looked through the issues. I see that it was just merged, I'll rebase my PR against master.